lib/Fakturoid/Client.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Fakturoid;
 
+use DateTime;
+
```

Comment:
@RiKap one more thing: this doesn't seem to be necessary. Do we really need this? If not, let's get rid of it. .)

Reply:
```suggestion
```